symbols/overlay29.yml

```diff
@@ -1371,6 +1390,13 @@ overlay29:
         Seems to be a function used during development to disable recruiting. If it returns false, SpecificRecruitCheck will also return false.
         
         return: true
+    - name: TryActivateIqBooster
+      address:
+        NA: 0x22EFA74
+      description: |-
+        If floor_properties::iq_booster_value is more than 0, increasces the IQ of all team members holding the IQ Booster.
```

Comment:
iirc, this increase is equal to the IQ booster value, right? If so, that should be specified here.

Reply:
(Still unresolved despite being marked as outdated, that was a typo fix)